scripts/Makefile.dtbo

```diff
@@ -7,18 +7,14 @@ __dtbo     := $(addprefix $(obj)/,$(__dtbo))
 dtbo-base	:= $(addprefix $(obj)/,$(dtbo-base))
 dtbo	:= $(addprefix $(obj)/,$(dtbo))
 
-ifneq ($(DTC_OVERLAY_TEST_EXT),)
-DTC_OVERLAY_TEST = $(DTC_OVERLAY_TEST_EXT)
+DTC_OVERLAY_TEST = scripts/ufdt_apply_overlay.bin
```

Comment:
```
  VERIFY  arch/arm64/boot/dts/qcom/trinket-seine-pdx201_generic-overlay.dtbo
/bin/sh: scripts/ufdt_apply_overlay.bin: No such file or directory
```

When building outside the tree it'll not find the script.

```suggestion
DTC_OVERLAY_TEST = $(srctree)/scripts/ufdt_apply_overlay.bin
```

Or just revert this thing again, it seems to be for testing only...